Software/Rig/Rig.ino

```diff
@@ -233,6 +236,14 @@ void simulateButtonPress() {
   // sensitive operations!!!
   delay(SECONDARY_PIN_PAUSE);
   digitalWrite(PIN_SECONDARY_1, HIGH);
+
+  if (!IS_PRIMARY_RIG) {
+    if (digitalRead(PIN_BUTTON) == HIGH) {
+      print("Connected to primary rig");
+    } else {
+      print("Not seeing connection to primary rig");
+    }
+  }
```

Comment:
Not sure how helpful it is but with the rigs daisy chained, the button should read as high (this is also true when the button is connected, though)